NSS-Day-Cohort-42/rare-API-media-moguls
# Fixed image name on rare user model

#### Changes Made
1. Modified file `rareuser.py` in `models` dir to rename profile image field.
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout mp-fix-img
```
2. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
